src/requests/process-paths.ts

```diff
@@ -61,6 +63,38 @@ function getSimpleName(url: string) {
   return method;
 }
 
+/**
+ * Returns name of the method
+ * @param method
+ */
 function getName(method: Method) {
   return _.upperFirst(_.camelCase(method.tags[0].replace(/(-rest)?-controller/, '')));
 }
+
+/**
+ * One of the allowed swagger formats is that under given url, there can be methods like get, post, put etc., but also
+ * parameters that often defines a path parameter common for the HTTP methods.
+ * This method extends HTTP method (get, post ...) parameters with the above mentioned parameters
+ * @param paths
+ */
+function preProcessPaths(paths: Paths): Paths {
+  Object.entries(paths).forEach(([pathKey, pathValue]) => {
+    if (pathValue.parameters) {
+      // deep copy is necessary
+      const methodsParameters = JSON.parse(JSON.stringify(pathValue.parameters));
+      delete pathValue.parameters;
+
+      Object.entries(pathValue).forEach(([methodName, method]) => {
+        try {
+          for (const met of methodsParameters) {
```

Comment:
Instead of this one a one-line construct
`a = a.concat(b)` or
`Array.prototype.push.apply(a, b)`
can be used.

Reply:
The `a.concat(b)` syntax was used.